ehrql/tables/beta/core.py

```diff
@@ -97,13 +108,11 @@ class ons_deaths(PatientFrame):
 
     date = Series(
         datetime.date,
-        description=(
-            "Patient's date of death. "
-            "Only deaths registered from February 2019 are recorded."
```

Comment:
I thought it'd be better to highlight this in the main documentation so it's easier to see.